### PR TITLE
LVMCluster CRD changes

### PR DESCRIPTION
**Pull request by nbalacha**
The LVMCluster.Spec has been updated to add a "storage" field which will contain storage related configuration.

### CODE REVIEW

File: api/v1alpha1/lvmcluster_types.go [hunk 1]
```diff
@@ -31,9 +31,9 @@ type LVMClusterSpec struct {
 	// Tolerations to apply to nodes to act on
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
-	// DeviceClasses are a rules that assign local storage devices to volumegroups that are used for creating lvm based PVs
+	// Storage describes the deviceClass configuration for local storage devices
```
Comment (Yuggupta27) [inline api/v1alpha1/lvmcluster_types.go:34]: To keep it future-ready, should we update the description of the `Storage` field to be more generic or less aligned with `DeviceClass`; Thoughts?

Comment (nbalacha) [reply api/v1alpha1/lvmcluster_types.go:34]: A good point. How about "Storage describes the configuration for local storage devices" ?

Comment (Yuggupta27) [reply api/v1alpha1/lvmcluster_types.go:34]: Sounds Great!

**Review by sp98 — APPROVED** [review]

**Review by leelavg — APPROVED** [review]

**Pull request merged**